### ISSUE TITLE
isBot v4 breaks remix dev with a fresh project

### ISSUE THREAD

**Issue by Nayshlok** [remix-run/remix#8434]
### Reproduction

Setup a new Remix app as detailed in the Quick Start https://remix.run/docs/en/main/start/quickstart
use `npx remix dev`

### System Info

```shell
System:
    OS: macOS 12.2
    CPU: (10) arm64 Apple M1 Max
    Memory: 17.90 GB / 64.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 20.10.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.2.5 - /usr/local/bin/npm
    pnpm: 8.6.6 - /usr/local/bin/pnpm
  Browsers:
    Chrome: 120.0.6099.129
    Safari: 15.3
  npmPackages:
    @remix-run/dev: ^2.4.1 => 2.4.1 
    @remix-run/node: ^2.4.1 => 2.4.1 
    @remix-run/react: ^2.4.1 => 2.4.1 
    @remix-run/serve: ^2.4.1 => 2.4.1
```

### Used Package Manager

npm

### Expected Behavior

remix run will start the server without a problem

### Actual Behavior

```
import isbot from "isbot";
       ^^^^^
SyntaxError: The requested module 'isbot' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:132:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:214:5)
    at ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at importModuleDynamicallyWrapper (node:internal/vm/module:431:15)
    at run (/Users/david.borland/code/playground/invoice-mfe/node_modules/@remix-run/serve/dist/cli.js:111:15)
```

**Comment by omrilotan** [comment]
recent [change](https://github.com/remix-run/remix/pull/8415) handles this.

If you can, change to the named import

```diff
- import isbot from "isbot";
+ import { isbot } from "isbot";
``

**Comment by brophdawg11** [comment]
Dup of #8407, #8392

* If you have `app/entry.server.tsx`, you can change the import to a named import as mentioned above
* If you are using the built-in versions - just `npm install isbot@3` to resolve the issue

I updated the quickstart docs a few minutes ago to change the install to isbot@3 until the next release when this is resolved (#8415)
